detect-monero-demo/README.md

```diff
@@ -0,0 +1,57 @@
+# Detect Monero Miners with bpftrace
+This demo accompanies the [Detect Monero Miners with bpftrace blogpost]().
+
+## Deploying xmrig to your cluster
+**Cloud Providers have strict policies around cryptomining. We strongly recommend not deploying
```

Comment:
Should we just add an optional prerequisite stage to use k0s at the beginning here?